pkg/http/api_test.go

```diff
@@ -153,6 +209,36 @@ func TestV1OutputBindingsEndpoints(t *testing.T) {
 		}
 	})
 
+	t.Run("Invoke output bindings - 200 OK", func(t *testing.T) {
+		apiPath := fmt.Sprintf("%s/bindings/testresponse", apiVersionV1)
+		req := OutputBindingRequest{
+			Data: "fake output",
+		}
+		b, _ := json.Marshal(&req)
+		testMethods := []string{"POST", "PUT"}
+		for _, method := range testMethods {
+			// act
+			resp := fakeServer.DoRequest(method, apiPath, b, nil)
+			// assert
+			assert.Equal(t, 200, resp.StatusCode, "failed to invoke output binding with %s", method)
```

Comment:
200 should be fasthttp.StatusOK and below too

Reply:
In this scenario it is more readable since most of the test names consist of status codes rather than constants. So I would prefer to leave it as such for tests. But not in the actual code.